### DISCUSSION TITLE
【视频】编码测试

### DISCUSSION THREAD

**Discussion by capacitor1** [share121/inter-knot#1083] [其他]
## 检测设备对不同编码的支持性
能播放为支持，不能播放为不支持。（也有可能是没加载出来

mpeg *比较老的编码，老视频上面常见*

https://github.com/user-attachments/assets/d135b513-cdf5-46cb-9277-f0a06c6c81b0

mjpeg *接近无损的编码，稍微损了一点点*

https://github.com/user-attachments/assets/e0607b9b-e0ca-4aaf-997c-9efe58491aef

vp9 *YouTube视频编码，米哈游usm格式CG的编码*

https://github.com/user-attachments/assets/103a5857-7626-4265-bfbf-9cfe2b1f7e87

avc *最常见的264*

https://github.com/user-attachments/assets/e536ee10-ecd9-4cda-82c0-a3465b8152c8

hevc *最常见的265*

https://github.com/user-attachments/assets/276664a8-ee8a-48c6-a514-20533225593b

**Comment by ijixng** [comment]
什么鬼？

**Comment by xiubachi** [comment]
我这观测不到

**Comment by guanshanyyy** [comment]
前两个不行，其他的可以

**Comment by SixRadium** [comment]
hevc是需要自己加插件的

**Comment by themagicmoon** [comment]
就vp9可以😭

**Comment by Ts-yf** [comment]
我这能看第三个和第四个

**Comment by luo-luo-o** [comment]
我这边第一、第二个不能播放，后面几种格式可以播放但加载缓慢（不确定是否为我自己的网络问题）
